rs/bitcoin/kyt/BUILD.bazel

```diff
@@ -10,9 +24,27 @@ rust_canister(
     proc_macro_deps = ["@crate_index//:ic-cdk-macros"],
     service_file = "btc_kyt_canister.did",
     deps = [
-        "@crate_index//:bitcoin",
+        ":btc_kyt",
+        "@crate_index//:bitcoin_0_32",
         "@crate_index//:candid",
         "@crate_index//:futures",
         "@crate_index//:ic-cdk",
     ],
 )
+
+rust_ic_test(
+    name = "integration_test",
+    srcs = ["tests/tests.rs"],
+    data = [":btc_kyt_canister"],
+    env = {
+        "CARGO_MANIFEST_DIR": "rs/bitcoin/kyt",
+        "IC_BTC_KYT_CANISTER_WASM_PATH": "$(rootpath :btc_kyt_canister)",
+    },
+    deps = [
+        ":btc_kyt",
+        "//rs/state_machine_tests",
```

Comment:
Since Pocket IC is the new cool kid in town and I think the official recommendation is to write new tests with Pocket IC and migrate existing ones to Pocket IC, shouldn't we use Pocket IC directly?

Reply:
That's a good point.

Reply:
Thanks! Will look into that